assessments/0009-iin-toto.md

```diff
@@ -0,0 +1,292 @@
+
+**High-level technical overviews** of increasing depth are needed, addressed to specific audiences; evaluators, new users, experienced users, contributors of different types (ITE proposers, doc writers and editors...). 
+- A basic intro, possibly suitable for evaluators, is already surfaced directly on the [home page About tab](https://in-toto.io/in-toto/). 
+   The About page has a link to the [latest spec in GitHub](https://github.com/in-toto/docs/blob/master/in-toto-spec.md), which contains a more comprehensive overview. This intro should immediately be surfaced as a Technical Overview document in RTD. It could be linked to the basic one as "Read more...".
+   (The [PDF link to the stable spec](https://github.com/in-toto/docs/blob/v1.0/in-toto-spec.pdf) is broken. This should be fixed or removed.)
+  
+- Determine whether the Tech Overview currently in the spec is suitable for new users or for contributors, and adapt or create another version of suitable depth for the other audience. The most in-depth overview should also point to academic papers for further architectural detail:
+https://www.usenix.org/system/files/sec19-torres-arias.pdf 
+https://www.usenix.org/conference/usenixsecurity19/presentation/torres-arias 
+
+- The new Doc home page should probably be the landing page for the [read-the-docs site](https://in-toto.readthedocs.io/en/latest/), which currently lands on the auto-generated Python reference doc. Tech overviews addressed to specific audiences should be added to RTD as text-only pages, organized by intended audience. 
+
+**Reference docs** for different implementations
+
+- Decide where reference docs for different implementations should live in the doc structure and where their sources live in the repo structure.
+
+- The doc roadmap should clearly identify and link to the existing generated pages as *reference* doc for the Python reference implementation.
+
+- The doc roadmap should also list and link reference docs for other implementations, such as auto-generated docs for the Go implementation: https://pkg.go.dev/github.com/in-toto/in-toto-golang. (This doc is not at all parallel with the Python RTD reference doc - it is all in GitHub, and has no introductory or explanatory content, or navigational aids.)
+
+- As noted below, policy and procedures for documenting implementations need to be decided and published for contributors.
+
+**New user content**
+
+- The [README for the main repo](https://github.com/in-toto/in-toto) looks like the intended starting place. This content should be surfaced as a separate **"Getting Started" document**, with a prominent pointer on the home page, such as the top menu item in the "Get Started" menu.
+
+We recommend moving all of this the doc to a documentation platform such as RTD. Put some brief introductory notes in the README that points the user to the documentation.
```

Comment:
We should clarify what we mean by platform.

Generally the TechDocs team suggests a stack of:
- Hugo
- Netlify
- Docsy

I think that read the docs fits here, but we should try to figure out how it all works together in an architecture suggestion.